InterGalacticCurrencyConvertor/src/main/java/DataProcessResult.java

```diff
@@ -0,0 +1,44 @@
+package main.java;
+
+public class DataProcessResult{
+	private String mOutput;
+	private Boolean mSuccess;
+
+	public DataProcessResult () {
+		mOutput = null;
+		mSuccess = false;
+	}
+	public void setOutput(String inOutput) {
+		this.mOutput = inOutput;
+	}
+	public void setSuccess(Boolean inSuccess) {
+		this.mSuccess = inSuccess;
+	}
+	public String getOutput() {
+		return mOutput;
+	}
+	public Boolean hasSucceeded() {
+		return mSuccess;
+	}
+	@Override
+	public boolean equals(Object inObj) {
```

Comment:
You haven't implemented hashCode() method.

Reply:
updated